support/recompile_check.py

```diff
@@ -27,24 +59,28 @@
     'p2_puzzle_hash.clsp',
     'p2_singleton.clsp',
     'p2_singleton_or_delayed_puzhash.clsp',
-    'pool_member_innerpuz.clsp',
-    'pool_waitingroom_innerpuz.clsp',
-    'rom_bootstrap_generator.clsp',
     'settlement_payments.clsp',
     'sha256tree_module.clsp',
     'singleton_launcher.clsp',
     'singleton_top_layer.clsp',
     'singleton_top_layer_v1_1.clsp',
-    'test_generator_deserialize.clsp',
-    'test_multiple_generator_input_arguments.clsp'
+    gentest('test_generator_deserialize.clsp'),
+    gentest('test_multiple_generator_input_arguments.clsp')
 ]
 
 for fname in recompile_list:
-    hexfile = f'./chia/wallet/puzzles/{fname}.hex'
+    if 'dirname' in fname and 'fname' in fname:
```

Comment:
these variables could use better names

Reply:
I used nicer names that are more indicative.